frontend/www/js/omegaup/arena/arena.test.ts

```diff
@@ -118,10 +130,42 @@ describe('arena', () => {
           new window.URL('http://localhost:8001/arena/test/'),
         ),
       );
-      arenaInstance.problemsetLoaded({
-        start_time: new Date(serverTime),
-        finish_time: new Date(serverTime + 3600),
+      arenaInstance.problems = {
+        test: {
+          accepts_submissions: true,
+          alias: 'test',
+          commit: 'abcdef',
+          input_limit: 10240,
+          languages: ['py2', 'py3'],
+          points: 0,
+          quality_seal: true,
+          title: 'Test',
+          visibility: 2,
+        },
+      };
+      arenaInstance.currentProblem = {
+        accepts_submissions: true,
+        title: 'Test',
+        alias: 'test',
+        commit: 'abcdef',
+        source: 'omegaUp',
+        languages: ['py2', 'py3'],
+        points: 0,
+        input_limit: 10240,
+        quality_seal: true,
+        visibility: 2,
+      };
+      arenaInstance.runSubmitView = new Vue({
+        data: () => ({
+          languages: ['py2', 'py3'],
+          preferredLanguage: 'py3',
+          nextSubmissionTimestamp: serverTime,
+        }),
       });
+      await arenaInstance.submitRun('print(3)', 'py3');
+      expect(arenaInstance.runSubmitView.$data.nextSubmissionTimestamp).toEqual(
+        serverTime + 60,
```

Comment:
```suggestion
        time + 60,
```

welp, esto no está bien. aquí esto debería aseverar que esto está manipulando correctamente los tiempos, pero parece que no es el caso? Según yo L162 también debería ser `now` y no `serverTime` (onda, `serverTime` sólo se puede usar en los APIs mockeados. fuera de ahí, todo debería ser basado en `now`).

Reply:
Listo. El problema es que no sé como hacer que se actualice `arenaInstance.currentProblem.nextSubmissionTimestamp`, ya que no es una promesa, el `await` no me funciona.